index.html

```diff
@@ -0,0 +1,147 @@
+              </li>
+            <li class="nav-item">
+              <a class="nav-link"  href="#">News</a>
+            </li>         
+            <li class="nav-item GE">
+                <a class="nav-link" id="ge" href="#">Campaign</a>
+              </li>    
+          </ul>
+          </div>
+    </navbar>
+   <!--Header-->
+   <header>
+    <h1 class="heading1">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
+    <p  class="parag">Welcome to our global annual summit that will be held virtually for ease of accessibility at your comfort.
+        For those who want to attend in person, it will be held at Uhuru Park, Nairobi,Kenya. </p>
+    <!-- Today's date -->
+    <h3 class="date"></h3>
+    <p class="sub">@ National Museum of Korea, Art Center and more</p>
+   </header>
+
+   <!-- Main program section -->
+   <section class="main-program">
+    <h3 class="heading3">Main Program</h3>
+    <hr class="hr">
+    <div class="grid-section">
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/save-water.png" alt="save-water"/>
+                <p class="first-p">Water</p>
+                <p class="second-p">We will discuss the benefits of avoiding water wastage.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/energy.png" alt="clean-energy"/>
+                <p class="first-p">Energy</p>
+                <p class="second-p">Exploration of energies that do not pollute environment such as biogas.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/plant.png" alt="conserve plants"/>
+                <p class="first-p">Forest</p>
+                <p class="second-p">Planting trees is very important in conservation of animal habitat and others that will be discussed.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/light.png" alt="Knowledge"/>
+                <p class="first-p">Knowledge</p>
+                <p class="second-p">Importance of teaching our young ones the benefit of conserving environment.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/recycle.png" alt="Recycle"/>
+                <p class="first-p">Recycle</p>
+                <p class="second-p">Plastic waste contribute to a huge percent of sea animals suffocation.</p>
+            </div>
+        </div>
+        <div class="grid-item2">
+            <h3 class="join-p">Join Green Earth Global Summit</h3>
+        </div>
+
+    </div>
+    <h3 class="heading31">SEE THE WHOLE PROGRAM</h3>
+    <hr class="hrprog">
+   </section>
+
+   <!-- Featured speakers -->
+   <section class="Featured">
```

Comment:
- [ ] I kindly suggest to consider removing the `square` like background image from this section of the speakers. This is to ensure that you match the original design provide ☺️.

Your design 😎 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/dde54f9b-d83f-4076-b715-f68fba8b1f7c)

Original design 😎 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/0f953659-9b50-45bf-b969-c81936928eb3)